pallets/gear/src/manager/task.rs

```diff
@@ -132,21 +136,57 @@ where
 
     fn send_dispatch(&mut self, stashed_message_id: MessageId) {
         // No validation required. If program doesn't exist, then NotExecuted appears.
-        let dispatch = DispatchStashOf::<T>::take(stashed_message_id)
+
+        let (dispatch, stored_bn) = DispatchStashOf::<T>::take(stashed_message_id)
             .unwrap_or_else(|| unreachable!("Scheduler & Stash logic invalidated!"));
 
+        // Unlocking gas for delayed sending rent payment.
+        GasHandlerOf::<T>::unlock_all(dispatch.id())
+            .unwrap_or_else(|e| unreachable!("GasTree corrupted! {:?}", e));
+
+        // Charging locked gas for holding in dispatch stash
```

Comment:
Dots at the end of each comment everywhere

Reply:
Ok